core-features/tracing/run.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+set -eux
+
+yarn install
+yarn prisma generate
+
+docker-compose up -d
```

Comment:
Could you move this to a new `prepare.sh` file? It will be executed automatically by the setup first.